sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -59,7 +59,15 @@ def __init__(self, is_embedded_fnapi_runner=False):
 
   @staticmethod
   def default_docker_image():
-    if 'USER' in os.environ:
+    if 'DOCKER_IMAGE' in os.environ:
```

Comment:
This is already controlled by the flag `--harness_docker_image`:
https://github.com/apache/beam/blob/385faa713951813371dffaf654b5dc8d96e27aa1/sdks/python/apache_beam/options/pipeline_options.py#L648

Do you still want to make the default container selection be based off of `DOCKER_IMAGE`?
If yes, should it specify the full path and not assume the user wants the `:latest` suffix?

Reply:
ah, yea, I just saw the pipeline option for this as well! thanks for pointing it out.

Reply:
(I'll revert this part of the change, I don't think it's necessary)